src/components/startups/Welcome.vue

```diff
@@ -1,7 +1,10 @@
 <template>
-    <startup-layout ref="layout" class="kiwi-welcome-simple">
+    <startup-layout ref="layout" class="kiwi-welcome-simple" >
         <template v-slot:connection v-if="!network || network.state === 'disconnected'">
-            <form class="u-form kiwi-welcome-simple-form" @submit.prevent="formSubmit">
+            <form :class="{ 'kiwi-recaptcha-enabled': recaptchaSiteId }"
```

Comment:
This class would be cleaner in the top most element just in case anything else needs adjusting in future. ie. `kiwi-welcome-simple--recaptcha`